### ISSUE TITLE
Wildcard search issue

### ISSUE THREAD

**Issue by compfix** [tjake/Solandra#69]
We have been running into solr search issues not returning the correct information.  I have added some additional test cases to SolandraTests and confirmed that there is an issue in the latest version of the solandra branch.

The tests are below.  The first and 3rd test cases pass but the second one does not.  All of the queries should return the same number of docs.

Thanks,

Cliff

@Test
    public void testWildCardQuery1() throws Exception
    {

```
    SolrQuery q = new SolrQuery().setQuery("*:*");

    try {
        QueryResponse r = otherClient.query(q);
        System.out.println("Results: " + r.getResults().getNumFound());
        assertEquals(42, r.getResults().getNumFound());
    } catch (Exception ex) {
        ex.printStackTrace();
    }

}

@Test
public void testWildcardQuery2() throws Exception
{

    SolrQuery q = new SolrQuery().setQuery("messageType:[* TO *]");

    try {
        QueryResponse r = otherClient.query(q);
        System.out.println("Results: " + r.getResults().getNumFound());
        assertEquals(42, r.getResults().getNumFound());
    } catch (Exception ex) {
        ex.printStackTrace();
    }

}

@Test
public void testWildcardQuery3() throws Exception
{

    SolrQuery q = new SolrQuery().setQuery("uuid:[* TO *]");

    try {
        QueryResponse r = otherClient.query(q);
        System.out.println("Results: " + r.getResults().getNumFound());
        assertEquals(42, r.getResults().getNumFound());
    } catch (Exception ex) {
        ex.printStackTrace();
    }

}
```

**Comment by tjake** [comment]
I can't reproduce with your test

**Comment by tjake** [comment]
This was reproducible by re-ordering the tests, turns out it was a edge case when searching with wildcards. now fixed in c04e2ac3edd5720cb0fe8be6c52d1093aac5dfa9
